foreman/data_refinery_foreman/surveyor/sra.py

```diff
@@ -379,13 +391,22 @@ def _generate_experiment_and_samples(self, run_accession: str) -> None:
                          survey_job=self.survey_job.id)
         except Sample.DoesNotExist:
             sample_object = Sample()
+            sample_object.source_database = "SRA"
             sample_object.accession_code = sample_accession_code
             sample_object.organism = organism
 
-            sample_object.platform_name = metadata.get("platform_instrument_model", "No model.")
+            sample_object.platform_name = metadata.get("platform_instrument_model", "UNKNOWN")
             # No platform accession nonsense with RNASeq, just use the name:
             sample_object.platform_accession_code = sample_object.platform_name
             sample_object.technology = "RNA-SEQ"
+            if "ILLUMINA" in sample_object.platform_name.upper():
+                sample_object.manufacturer = "ILLUMINA"
+            elif "NEXTSEQ" in sample_object.platform_name.upper():
```

Comment:
NextSeq is an Illumina product: https://www.illumina.com/systems/sequencing-platforms/nextseq.html

Not exactly sure why that is not reflected in the platform name from data sources. Is the distinction important here or should this also be `sample_object.manufacturer = "ILLUMINA"`?

Reply:
It is not important (at least not yet). At this point manufacturer is pretty much ignored for RNA-SEQ technologies, I just figured that since we could we may as well record it now in case we ever did have a use for it later. I will update this so NEXTSEQ gets marked as  having ILLUMINA for a manufacturer.